src/dailies.ts

```diff
@@ -557,7 +564,12 @@ export function hipsterFishing(): void {
     const runSource = findRun(false);
     if (!runSource) return;
     if (runSource.prepare) runSource.prepare();
-    freeFightOutfit([...(runSource.requirement ? [runSource.requirement] : [])]);
+    freeFightOutfit([
+      ...(runSource.requirement ? [runSource.requirement] : []),
+      ...(have($familiar`Mini-Hipster`)
+        ? [new Requirement([], { forceEquip: [bestHipsterEquip] })]
```

Comment:
passing all 3 with a bonus may be better in case any of the equipment isn't available it will just handle it

Reply:
You're right, that's way better, implemented.